vtex-apps/shelf
# Use store-resources query in Shelf

#### What problem is this solving?

We forget to add new fields in this app. Moving the queries to store-resources this issue is solved.

#### How to test it?

Check the Shelf at the home page:
https://brenovtex--sarachoneumaticos.myvtex.com/

Check the Shelf Recommendation at the bottom of the PDP:
https://brenovtex--sarachoneumaticos.myvtex.com/215-65-r16-michelin-primacy-4-102h/p

#### Screenshots or example usage:

n/a

#### Describe alternatives you've considered, if any.

n/a

#### Related to / Depends on

Depends on https://github.com/vtex-apps/store-resources/pull/132

#### How does this PR make you feel? [:link:](http://giphy.com/)



![](https://media.giphy.com/media/3o7WTHbVK7BNBOPBnO/giphy.gif)
